### ISSUE TITLE
can not install 0.5.1 bee

### ISSUE THREAD

**Issue by iljooyoon** [ethersphere/bee#1335]
i can upgrade bee from 0.5.0 to 0.5.1.

but if i install without previous version of bee, it stops

root@Bee09:~# sudo dpkg -i bee_0.5.1_amd64.deb
Selecting previously unselected package bee.
(Reading database ... 143695 files and directories currently installed.)
Preparing to unpack bee_0.5.1_amd64.deb ...
Unpacking bee (0.5.1) ...
Setting up bee (0.5.1) ...

**Comment by XO39** [comment]
I'm having the same issue on Centos 8, can't even revert back to the older version with older data

**Comment by vandot** [comment]
@iljooyoon can you please explain what `if i install without previous version of bee, it stops` means?
what stops?

**Comment by vandot** [comment]
@XO39
- what files were in conflict?
- we explicitly check if that folder exists and create if not https://github.com/ethersphere/bee/blob/master/packaging/rpm/pre#L9
- please collect logs with `sudo journalctl -f -u bee`

**Comment by XO39** [comment]
@vandot Sorry, I don't really remember those file, after more than ten retries to make work, at the end I had to downgrade to v0.5.0 get it working again, unfortunately I had to start over without the backup files, that means new ethereum address and all. :(

**Comment by XO39** [comment]
But since I just started over from scratch, I'll try to do the upgrade again and report again here in this issue.

**Comment by XO39** [comment]
Here is what I've tried so far:
With older version of both `bee-clef (v0.8.7)` and `bee (v0.5.0)` installed, I tried the following:

- `sudo rpm -i bee-clef_0.4.8_amd64.rpm`, got the following errors:
 
  ```
 	file /usr/bin/bee-clef-keys from install of bee-clef-0:0.4.8-1.x86_64 conflicts with file from package bee-clef-0:0.4.7-1.x86_64
 	file /usr/bin/bee-clef-service from install of bee-clef-0:0.4.8-1.x86_64 conflicts with file from package bee-clef-0:0.4.7-1.x86_64
 	file /usr/bin/clef from install of bee-clef-0:0.4.8-1.x86_64 conflicts with file from package bee-clef-0:0.4.7-1.x86_64
   ```

-   Then: `sudo rpm -U bee-clef_0.4.8_amd64.rpm `, this worked.

-   And then I did `sudo rpm -i bee_0.5.1_amd64.rpm`, failed with the following errors:
   ```
 	file /etc/bee/bee.yaml from install of bee-0:0.5.1-1.x86_64 conflicts with file from package bee-0:0.5.0-1.x86_64
 	file /usr/bin/bee from install of bee-0:0.5.1-1.x86_64 conflicts with file from package bee-0:0.5.0-1.x86_64
 	file /usr/bin/bee-get-addr from install of bee-0:0.5.1-1.x86_64 conflicts with file from package bee-0:0.5.0-1.x86_64
   ```
 
-   Then I did `sudo rpm -U bee_0.5.1_amd64.rpm` this finally works, but `sudo bee-get-addr` stopped working, and it just hangs with no output at all.

  Then I just removed every thing and started over from scratch:


  

- `sudo rpm -i bee-clef_0.4.8_amd64.rpm` no issue
-   `sudo rpm -i bee_0.5.1_amd64.rpm` this didn't work, just hangs forever and I had to cancel it with Ctrl-C, and then get this error `chown: cannot access '/var/lib/bee/keys': No such file or directory`, but also says it already installed, and the `/var/lib/bee/`/ only contains the `password` file and nothing else.
-   `sudo yum remove -y bee` I had to remove it, then;
-   `sudo rpm -i bee_0.5.0_amd64.rpm` this works, then;
-   `sudo rpm -U bee_0.5.0_amd64.rpm` this also works, but again, `sudo bee-get-addr` stopped working, and it just hangs with no output at all.
- After I start the `bee` service and check the status, it just doesn't work, I only get one line of log: `time="xxxx-xx-xx xx:xx:xx" level=info msg="version: 0.5.1-65ec2d0"` and nothing else, and the log is also the same. But if I set `clef-signer-enable` to `false`, I then get output, but this time with this warning: `cannot continue until there is sufficient ETH (for Gas) available on xx...`, althogh I've funded the account from the Disocred `#faucet-channel`.

**Comment by iljooyoon** [comment]
> @iljooyoon can you please explain what `if i install without previous version of bee, it stops` means?
> what stops?
Installation has stopped. Please understand that English is poor.

**Comment by vandot** [comment]
@XO39 thank for the report, I checked it now, you are right for `rpm -i` and  `rpm -U` we will change it inside docs to `yum localinstall` you use same command for install and upgrade, after the upgrade bee and bee-clef should be started but they are not, I will fix this
@iljooyoon new/fresh install doesn't automatically start the service, because you first have to fund the node etc, after fresh install you have to `sudo systemctl start bee`

for both main issues were caused by bad bee-clef 0.4.8, its fixed in bee-clef 0.4.9

**Comment by iljooyoon** [comment]
> @XO39 thank for the report, I checked it now, you are right for `rpm -i` and `rpm -U` we will change it inside docs to `yum localinstall` you use same command for install and upgrade, after the upgrade bee and bee-clef should be started but they are not, I will fix this
> @iljooyoon new/fresh install doesn't automatically start the service, because you first have to fund the node etc, after fresh install you have to `sudo systemctl start bee`
> 
> for both main issues were caused by bad bee-clef 0.4.8, its fixed in bee-clef 0.4.9

hm... "stop" means.. 
if i type "root@Bee09:~# sudo dpkg -i bee_0.5.1_amd64.deb"
then install logs appear, and stop.

prompt doesn't appear


[error]
root@Bee09:~# sudo dpkg -i bee_0.5.1_amd64.deb
Selecting previously unselected package bee.
(Reading database ... 143695 files and directories currently installed.)
Preparing to unpack bee_0.5.1_amd64.deb ...
Unpacking bee (0.5.1) ...
Setting up bee (0.5.1) ...


[normal]
root@Bee09:~# sudo dpkg -i bee_0.5.1_amd64.deb
Selecting previously unselected package bee.
(Reading database ... 143695 files and directories currently installed.)
Preparing to unpack bee_0.5.1_amd64.deb ...
Unpacking bee (0.5.1) ...
Setting up bee (0.5.1) ...
~~  maybe some more logs.. ~~
root@Bee09:~#

**Comment by iljooyoon** [comment]
This bug was fixed in bee-clef 0.4.9.
